packages/shared/src/components/filters/FollowingFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { type ReactElement } from 'react';
+import Link from 'next/link';
+import { useAuthContext } from '../../contexts/AuthContext';
+import UserList from '../profile/UserList';
+import { useFollowingQuery } from '../../hooks/contentPreference/useFollowingQuery';
+import { ContentPreferenceType } from '../../graphql/contentPreference';
+import { FlexCentered } from '../utilities';
+import { checkFetchMore } from '../containers/InfiniteScrolling';
+import { AddUserIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { useLazyModal } from '../../hooks/useLazyModal';
+
+export const FollowingFilter = (): ReactElement => {
+  const { user } = useAuthContext();
+  const { closeModal } = useLazyModal();
+  const queryResult = useFollowingQuery({
+    id: user.id,
+    entity: ContentPreferenceType.User,
+  });
+  const { data, isFetchingNextPage, fetchNextPage } = queryResult;
+  const users = data?.pages.reduce((acc, p) => {
+    p?.edges.forEach(({ node }) => {
+      acc.push(node.referenceUser);
+    });
+
+    return acc;
+  }, []);
+
+  return (
+    <UserList
+      users={users}
+      emptyPlaceholder={
+        <FlexCentered className="flex-col gap-4 px-6 py-10 text-center text-text-tertiary typo-callout">
+          <AddUserIcon size={IconSize.XXXLarge} />
+          <p>
+            You haven&apos;t follow any User yet.
+            <br /> Explore our{' '}
+            <Link
+              href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}users`}
+              passHref
+              onClick={() => {
+                closeModal();
+              }}
+              className="text-text-link"
+            >
+              Leaderboards
+            </Link>{' '}
+            to find and follow inspiring users!
+          </p>
+        </FlexCentered>
+      }
+      scrollingProps={{
+        isFetchingNextPage,
+        canFetchMore: checkFetchMore(queryResult),
+        fetchNextPage,
+      }}
+      userInfoProps={{
+        onClick: () => {
+          closeModal();
+        },
+      }}
```

Comment:
Same with this one, not sure why we need it?
(EG other modals (upvote list etc) don't do this right?

Reply:
Force close because modal is persist on navigation. But saw @sshanzel commented that it's probably legacy, so we can probably remove it and not have to use close-modal

Reply:
Yeah let's remove it, it's weird to have to do manual stuff basically.

Reply:
Removed and removed manual close.